core/frontend/src/components/scanner/servicesScanner.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <span />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import axios from 'axios'
+import { getModule } from 'vuex-module-decorators'
+import ServicesScannerStore from '@/store/servicesScanner'
+import {Service} from '@/types/SERVICE'
+
+const servicesHelper: ServicesScannerStore = getModule(ServicesScannerStore)
+
+/**
+ * Actual scanner for running services.
+ * This periodically fetches /helper/latest/web_services
+ * and updates the ServiceHelperStore with the available services
+ * @displayName Services Scanner
+ */
+export default Vue.extend({
+  name: 'ServicesFetcher',
+
+  mounted () {
+    // Fetch network data
+    this.requestData()
+
+    // Fetch periodic API request
+    this.startPeriodicRequest()
+  },
+  methods: {
+    startPeriodicRequest () {
+      setInterval(() => {
+        this.requestData()
+      }, 5000)
+    },
+    requestData () {
+      axios.get('http://localhost/helper/latest/web_services').then((response) => {
+        // Sort services by port number
+        const services = response.data.sort(
+          (first: Service, second: Service) => first.port - second.port,
+        )
+        servicesHelper.updateFoundServices(services)
+      }).catch((error) => {
+        console.log('Error scanning for services:')
+        // logging the error as a whole gives us a better backtrace
+        console.log(error)
+        servicesHelper.updateFoundServices([])
+      })
+    },
+  },
+})
+</script>
+
+<style scoped>
+.helper-table {
+    max-width: 70%;
+    margin: auto;
+}
+</style>
```

Comment:
This is not being used here, right?

Reply:
yes, it is, this just compresses the table horizontally a bit. It probably should be a media query, but I can't really do those.

Reply:
weird. I had some local changes I hadn't pushed yet. It is there now. line 2